db/schema.rb

```diff
@@ -129,6 +129,7 @@
     t.integer  "user_id",            limit: 4
     t.string   "commited_from",      limit: 255
     t.string   "last_commited_from", limit: 255
+    t.text     "referral_url",       limit: 65535
```

Comment:
I thought you had removed the limit due to the length issues with different versions of mysql?

Reply:
I had problem with to add an index on the text field but the text field length does not raise any problems. Since I end up not adding an index, then everything is fine! :)